setup.py

```diff
@@ -39,6 +39,7 @@
 EXTRAS['mujoco'] = [
     'mujoco-py<2.1,>=2.0',
     'gym[all]' + GYM_VERSION,
+    'metaworld @ git+https://github.com/rlworkgroup/metaworld.git@dfdbc7cf495678ee96b360d1e6e199acc141b36c',  # noqa: E501
```

Comment:
hmm as the maintainer of this package i'm way of putting this in a garage release. the package isn't very stable or even really packaged very much. at best i'd put this in a different extra like `benchmarks` or something.

Reply:
After some offline discussion, we decided it's fine to include it in `dev`, so that it can be used in tests for the time being.